<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogViewerClass</class>
 <widget class="QMainWindow" name="LogViewerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1059</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LogViewer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="LogViewer.qrc">
    <normaloff>:/imagens/ico</normaloff>:/imagens/ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,0,0,0,0,0,0,0,0,0">
      <item>
       <widget class="QLabel" name="labelProf">
        <property name="toolTip">
         <string>Nível de profundidade de filtragem de inspeção de registro </string>
        </property>
        <property name="text">
         <string>Depth &amp;level</string>
        </property>
        <property name="buddy">
         <cstring>spinBoxProf</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxProf"/>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>1</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>25</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="hs1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="labelAtual">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>of</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelTotal">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>kB</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="hs2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbPausa">
        <property name="text">
         <string>&amp;Pause</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbClean">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>&amp;Clear</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="lineWrapMode">
       <enum>QTextEdit::NoWrap</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1059</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Arquivo">
    <property name="title">
     <string>&amp;Data source</string>
    </property>
    <addaction name="actionAbrir"/>
    <addaction name="actionConectarServidor"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menu_Arquivo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <action name="actionAbrir">
   <property name="text">
    <string>Open &amp;File...</string>
   </property>
   <property name="iconText">
    <string>Abrir arquivo</string>
   </property>
   <property name="toolTip">
    <string>Abrir arquivo de dados de registro de aplicativo local</string>
   </property>
   <property name="statusTip">
    <string>Abrir arquivo de dados de registro de aplicativo local</string>
   </property>
   <property name="shortcut">
    <string>Alt+F</string>
   </property>
  </action>
  <action name="actionConectarServidor">
   <property name="text">
    <string>Conect to &amp;server...</string>
   </property>
   <property name="toolTip">
    <string>Conectar com servidor de registros de aplicativo remoto</string>
   </property>
   <property name="statusTip">
    <string>Conectar com servidor de registros de aplicativo remoto</string>
   </property>
   <property name="shortcut">
    <string>Alt+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Alt+X</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="LogViewer.qrc"/>
 </resources>
 <connections/>
</ui>
